### ISSUE TITLE
Unable to use mp4extract on atom names containing a .

### ISSUE THREAD

**Issue by eric-wieser** [axiomatic-systems/Bento4#397]
`mp4dump` gives me something like:
```
  [udta] size=8+154
    [meta] size=12+142
      [hdlr] size=12+21
        handler_type = mdir
        handler_name = 
      [ilst] size=8+101
        [.nam] size=8+56
          [data] size=8+48
            type = 1
            lang = 0
            value = <snip>
        [.too] size=8+29
          [data] size=8+21
            type = 1
            lang = 0
            value = Lavf57.83.100
```
`mp4extract moov/udta/ilst ...` works just fine, but `mp4extract moov/udta/ilst/.nam ...` complains that the atom is not found.

Is there something special about the syntax here for accessing atoms with dots in?

**Comment by eric-wieser** [comment]
Looks like something weird is going on with `mp4dump` and/or my stdio / editor here, and the spec reveals that the `.` is actually an `\xA9`

**Comment by barbibulle** [comment]
Yes, `mp4dump` replaces non-ascii characters with a `.` when printing the atom types. In this case the first character of the type is `©`, which is is 0xA9 in ISO 8859-1 encoding.
I should relax this restricting in `mp4dump` and allow printing non-ascii characters that are still expected to be printable on most terminals (i.e not control characters).

**Comment by eric-wieser** [comment]
It might be nice to print non-printable characters with `\xA9` or something to draw attention to them
